src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteWaypointModel.js

```diff
@@ -377,25 +373,25 @@ export default class StandardRouteWaypointModel extends BaseModel {
      * - "S210"
      *
      * @for StandardRouteWaypointModel
```

Comment:
can you verify the doc bloc example above?  is this still accurate with the additional properties?